manifests/init.pp

```diff
@@ -641,6 +641,9 @@
 #                                           Defaults to false
 #                                           type:Boolean
 #
+# $server_allow_header_cert_info::          Enable client authentication over HTTP Headers
+#                                           Defaults to false, and can also be activated with the $server_http setting
```

Comment:
the correct wording here is IMHO "..and is also activated by the...", but we can fix that on merge